src/snowflake/snowpark/modin/plugin/_internal/resample_utils.py

```diff
@@ -41,7 +41,17 @@
 
 SNOWFLAKE_TIMESLICE_ALIGNMENT_DATE = "1970-01-01 00:00:00"
 
-IMPLEMENTED_AGG_METHODS = ["max", "min", "mean", "median", "sum", "std", "var", "count"]
+IMPLEMENTED_AGG_METHODS = [
```

Comment:
This list in used in the tests, so `resample.size` is indeed being tested (despite no test changes in this PR).